docs/user_manual/components.md

```diff
@@ -985,6 +985,11 @@ See the documentation on [state estimation calculation methods](calculations.md#
 per method on how the variances are taken into account for both the global and local angle measurement types and for the
 individual phases.
 
+```{note}
+The combination of `i_measured=0` and `i_angle_measured=n&#960/2` renders the current sensor invalid for PGM. 
```

Comment:
unicode &#960 in code blocks is not supported. Instead, please use either latex


```suggestion
The combination of $i_measured=0$ and $i_angle_measured=\frac{n\pi}{2}$ renders the current sensor invalid for PGM. 
```

or plain text


```suggestion
The combination of `i_measured=0` and `i_angle_measured=n pi /2` renders the current sensor invalid for PGM. 
```

or remove the code block guard


```suggestion
The combination of `i_measured=0` and `i_angle_measured=n`&#960`/2` renders the current sensor invalid for PGM. 
```

```suggestion
The combination of i_measured=0 and i_angle_measured=n&#960/2 renders the current sensor invalid for PGM. 
```

Reply:
fixed in [40a91d2](https://github.com/PowerGridModel/power-grid-model/pull/1056/commits/40a91d27cbbe6d4789e85b245c6afda2f32d87bb)